api/src/services/es-service.ts

```diff
@@ -56,11 +56,28 @@ export class ESService {
 
       query: {
         multi_match: {
-          fields: ['project.taxonomicCoverage.commonName', 'project.taxonomicCoverage.taxonRankValue'],
+          fields: ['*'],
           type: 'phrase_prefix',
           query
         }
       }
     });
   }
+
+  /**
+   * Searches for projects based on a datasetId.
+   * @param query The species/keywords to search for
+   * @returns {Promise<SearchHit<unknown>>} The results of the search
+   */
+  async datasetSearchEml(datasetId: string): Promise<SearchHit<unknown>[]> {
```

Comment:
Can this be removed? Is it from earlier work fetching the meta from elastic search?